lib/grape/middleware/versioner/header.rb

```diff
@@ -86,6 +83,12 @@ def available_media_types
           available_media_types = available_media_types.flatten
         end
 
+        def rack_accept_header
+          Rack::Accept::MediaType.new env['HTTP_ACCEPT']
+          rescue RuntimeError => e
```

Comment:
This is mis-aligned I think, the `rescue` is usually at the `def` level to be a function-based rescue.